docs/manipulation.rst

```diff
@@ -85,6 +85,9 @@ that takes the spectrum and an astropy 1D kernel.  So, one could also do:
 In this case, the ``spec1_bsmooth2`` result should be equivalent to the ``spec1_bsmooth`` in
 the section above (assuming the flux data of the input ``spec`` is the same).
 
+The errors are propagated using a standrard "propagation of errors" method, if the uncertainty 
+is defined for the spectrum *and* it is one of StdDevUncertainty, VarianceUncertainty or InverseVariance.
+*Note:* this does *not* consider covariance between points.
```

Comment:
```suggestion
But note that this does *not* consider covariance between points.
```

Or if you want an actual note "box" (a la https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html#notes) , this could also be

```suggestion

.. note::
    This does *not* consider covariance between points.
    
```